scripts/build_glpk.py

```diff
@@ -55,7 +56,8 @@ def get_vcvarsall_cmd():
 os.chdir("%s/w%d" % (glpk_build_dir, bitness))
 if not os.path.isfile("glpk.lib"):
     shutil.copy2("config_VC", "config.h")
-    os.system(get_vcvarsall_cmd() + "& nmake /f Makefile_VC")
+    os.environ.update(setuptools.msvc.msvc14_get_vc_env(arch))
+    os.system("nmake /f Makefile_VC")
```

Comment:
Could use the opportunity to replace `os.system`

```suggestion
    subprocess.run(["nmake", "/f", "Makefile_VC"], check=True)
```